app/views/pages/history.html.erb

```diff
@@ -1,9 +1,5 @@
 <div class="page-container">
-  <%= stylesheet_link_tag "home" %>
-  <h1>History</h1>
-  <br>
-  <h4>Issues</h4>
-  <br>
+  <h1>Reported issues:</h1>
```

Comment:
Minor, but this should be (like the link), just "Reported Issues", with no colon and both words title case.

Reply:
ok